documentation/bolt_installing.md

```diff
@@ -161,6 +161,12 @@ Use the Apple Disk Image (DMG) to install Bolt on macOS:
 1. Double-click the `puppet-bolt-latest.dmg` file to mount the installer and
    then double-click `puppet-bolt-[version]-installer.pkg` to run the installer.
 
+If you get a message that the installer "can't be opened because Apple cannot check it for malicious software:"
+1. Click **** > **System Preferences** >> **Security & Privacy**.
```

Comment:
```suggestion
1. Click **** > **System Preferences** > **Security & Privacy**.
```

Not sure if the double bracket is intentional.

Reply:
Ah yep, that's a typo!